bundle/src/main/java/com/adobe/acs/commons/redirects/MyLocationAdjuster.java

```diff
@@ -0,0 +1,18 @@
+package com.adobe.acs.commons.redirects;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.osgi.service.component.annotations.Component;
+
+@Component
+public class MyLocationAdjuster implements LocationHeaderAdjuster{

```

Comment:
This implementation is not useful in the main code, please move to test classes

Reply:
removed from the code.